modules/client-configuration/pages/contact-methods-saltbundle.adoc

```diff
@@ -87,10 +90,14 @@ The Python 3 deployed with the Bundle is used to handle Salt SSH session on the
 [IMPORTANT]
 ====
 {productname} 4.2 is not using the Salt Bundle to for Salt SSH (including bootstrapping with {webui}) by-default.
-To enable the Salt Bundle with Salt SSH support the following parameter need to be specified in [path]``/etc/rhn/rhn.conf``:
+To enable the Salt Bundle with Salt SSH support the following parameter needs to be specified in [path]``/etc/rhn/rhn.conf``:
 ----
 web.ssh_salt_pre_flight_script = /usr/share/susemanager/salt-ssh/preflight.sh
 ----
+To prevent using salt thin on managing Salt SSH actions the following parameter needs to be specified in [path]``/etc/rhn/rhn.conf``:
+----
+web.ssh_use_salt_thin = false
+----
```

Comment:
I'd collapse both these params:

```suggestion
To enable the Salt Bundle with Salt SSH support and to disable so-called Salt thin, specify in [path]``/etc/rhn/rhn.conf``:
----
web.ssh_salt_pre_flight_script = /usr/share/susemanager/salt-ssh/preflight.sh

web.ssh_use_salt_thin = false
----
```
I'm not sure whether we must explain "Salt thin".  I can only guess what this is ;-)